ale/formatters/usgscsm_formatter.py

```diff
@@ -23,7 +23,7 @@ def to_usgscsm(driver):
     body_radii = driver.target_body_radii
     isd_data['radii'] = {
         'semimajor' : body_radii[0],
-        'semiminor' : body_radii[1],
+        'semiminor' : body_radii[2],
```

Comment:
??? Why thus change?

Reply:
This is due to what target_body_radii returns, specifically the spice call that is used to get the radii from the kernels. Its a set of three values the first being the maximum radius at the equator, the second being the minimum radius at the equator and the last being the polar radius.

If we want the semi major (equator) and semi minor (polar) then we need to get the first and third elements.